enigma/src/main/resources/lang/en_us.json

```diff
@@ -34,6 +34,7 @@
 	"menu.file.export.jar": "Export Jar...",
 	"menu.file.stats": "Mapping Stats...",
 	"menu.file.stats.title": "Mapping Stats",
+	"menu.file.stats.overall": "Overall",
```

Comment:
```suggestion
	"menu.file.stats.overall": "Overall Stats",
```
maybe?

Reply:
I mean you're in the "stats" window, I think it's inferred
if I changed that it would be inconsistent with the other texts

Reply:
👍up to you